docs/hub/models-downloading.md

```diff
@@ -37,9 +37,10 @@ model = joblib.load(
 
 ## Using Git
 
-Since all models on the Model Hub are Git repositories, you can clone the models locally by running:
+Since all models on the Model Hub are [Xet-backed](./xet/using-xet-storage#git-xet) Git repositories, you can clone the models locally by running:
```

Comment:
```suggestion
Since all models on the Model Hub are [Xet-backed](./xet/using-xet-storage#git-xet) Git repositories, you can clone the models locally by [installing git-xet](./xet/using-xet-storage#git-xet) and running:
```

Maybe, to be completely explicit?